src/documentation/pages/components/tabs/examples/_bordered_subtabs.vue

```diff
@@ -0,0 +1,96 @@
+<template lang="pug">
+  <ComponentTabExample title="Bordered" id="bordered-portal" padding="-p--0" :tabs="exampleTabs" :menuTabs="menuTabs" :menuId="menuId" @toggleMenuId="toggleMenuId">
+    .-p--3.-bg--grey-20(:slot="$data.menuTabs[0].id")
+      .-bg--white.-p--3
+        ul.chi-tabs.-border
+          li.-active
+            a(href='#') Active tab
+          li
+            a(href='#') Tab link
+          li
+            a(href='#') Tab link
+
+    .-p--3.-bg--grey-20(:slot="$data.menuTabs[1].id")
+      .-bg--black.-p--3
+        ul.chi-tabs.-inverse.-border
+          li.-active
+            a(href='#') Active tab
+          li
+            a(href='#') Tab link
+          li
+            a(href='#') Tab link
+    <pre class="language-html" slot="code-webcomponent">
+      <code v-highlight="$data.codeSnippets.webcomponent" class="html"></code>
+    </pre>
+    <pre class="language-html" slot="code-htmlblueprint">
+      <code v-if="menuId === $data.menuTabs[0].id" v-highlight="$data.codeSnippets.htmlblueprintBase" class="html"></code>
+      <code v-if="menuId === $data.menuTabs[1].id" v-highlight="$data.codeSnippets.htmlblueprintInverse" class="html"></code>
+    </pre>
+  </ComponentTabExample>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator';
+
+@Component({
+  data: () => {
+    return {
+      exampleTabs: [
+        {
+          disabled: true,
+          id: 'webcomponent',
+          label: 'Web component',
+        },
+        {
+          active: true,
+          id: 'htmlblueprint',
+          label: 'HTML blueprint',
+        },
+      ],
+         menuTabs: [
+        {
+          active: true,
+          id: 'base',
+          label: 'Base',
+        },
+        {
+          id: 'inverse',
+          label: 'Inverse',
+        },
+      ],
+      codeSnippets: {
+        webcomponent: ``,
+        htmlblueprintBase: `<ul class="chi-tabs -border">
+  <li class="-active">
+    <a href="#">Active tab</a>
+  </li>
+  <li>
+    <a href="#">Tab link</a>
+  </li>
+  <li>
+    <a href="#">Tab link</a>
+  </li>
+</ul>`,
+        htmlblueprintInverse: `<ul class="chi-tabs -inverse -border">
+  <li class="-active">
+    <a href="#">Active tab</a>
+  </li>
+  <li>
+    <a href="#">Tab link</a>
+  </li>
+  <li>
+    <a href="#">Tab link</a>
+  </li>
+</ul>`
+      },
+    };
+  },
+})
+export default class BorderedSubtabsLumenCenturyLink extends Vue {
+   menuId = 'base';
+
+  toggleMenuId(e: string) {
```

Comment:
is `e` an id? if so i'd name it id instead of e to make it more descriptive. In addition please run the formatter on this file as the indentation looks off

Reply:
Its an event actually, not an id.